testsuite/features/build_validation/migration/slemicro54_minion_migration.feature

```diff
@@ -59,13 +59,14 @@ Feature: Migrate a SLE Micro 5.4 Salt minion to SLE Micro 5.5
     And I reboot the "slemicro54_minion" minion through the web UI
     And I disable repositories after installing Salt on this "slemicro54_minion"
 
-  Scenario: Subscribe the SLE Micro minion to a base channel
+  Scenario: Subscribe the SLE Micro minion to SLE Micro 5.4 child channels
     Given I am on the Systems overview page of this "slemicro54_minion"
     When I follow "Software" in the content area
     And I follow "Software Channels" in the content area
     And I wait until I do not see "Loading..." text
-    And I check radio button "Test-Base-Channel-x86_64"
-    And I wait until I do not see "Loading..." text
+    And I should see the child channel "SLE-Manager-Tools-For-Micro5-Pool for x86_64 5.5" "unselected"
+    When I select the child channel "SLE-Manager-Tools-For-Micro5-Pool for x86_64 5.5"
+    Then I should see the child channel "SLE-Manager-Tools-For-Micro5-Pool for x86_64 5.5" "selected"
```

Comment:
Why do you use the SLEM 5.5 child channel here instead of the 5.4 ones?

Reply:
The channel is correct, as the minion is migrated from SLE Micro 5.4 to 5.5 in the previous scenarios.
What's wrong is this scenario name, thanks for the catch !

Reply:
Ok, that way it makes sense. From the scenario name, if confused me.